app/templates/application.hbs

```diff
@@ -1,37 +1,85 @@
-<div class="{{skin}} wrapper">
+<div class="container-fluid">
 
-<header class="main-header">
-  <!-- Logo -->
-  {{#link-to "index" class="logo"}}
-    <!-- mini logo for sidebar mini 50x50 pixels -->
-    <span class="logo-mini"><b>E</b> D</span>
-    <!-- logo for regular state and mobile devices -->
-    <span class="logo-lg"><b>Experimenter</b> Dashboard</span>
-  {{/link-to}}
+{{#if isNotLogin}}
+  <div class="row">
+    {{#if isExpanded}}
+      <div class="menu-expanded col-md-2">
+        <div>
+          <a class="fa fa-angle-left fa-pull-right fa-2x sub-navigation" {{action "toggleMenu"}}></a>
+        </div>
+        <div class="text-center logo-title title-divider row">
+          <div class="fa fa-flask fa-2x logo"></div>
+          Experimenter
+        </div>
+        {{#if isExperiments}}
+          <div class="row main-navigation selected">
```

Comment:
If you use Ember's link-to helpers it will manage the active state for you:
https://guides.emberjs.com/v2.3.0/templates/links/#toc_adding-additional-attributes-on-a-link